.settings/org.eclipse.core.resources.prefs

```diff
@@ -1,4 +1,5 @@
 eclipse.preferences.version=1
 encoding//src/main/java=UTF-8
+encoding//src/main/resources=UTF-8
```

Comment:
this file should entirely be removed from repository and everyting underneath `.settings/` directory